quartz/processors/parse.ts

```diff
@@ -140,17 +175,28 @@ export async function parseMarkdown(ctx: BuildCtx, fps: FilePath[]): Promise<Pro
       maxWorkers: concurrency,
       workerType: "thread",
     })
+    const errorHandler = (err: any) => {
+      const errString = err.toString().slice("Error:".length)
```

Comment:
This seems like a hack. If the err string doesn't start with `Error:` then the print would be mismatched.

--verbose might output different string downstream to plugins makers, so we won't know for sure

Reply:
we can just print out the error string fwiw no need to splice anything.

Reply:
I just moved the error callback that was already there to a variable (since now we need it twice) - not my hack 🙃

I didn't even think about the code that was in the callback :)

Anyway I'm in bed, will reply to all reviews here and in other places tmrw

Reply:
ok the splice was bad, fixed it ig

I do agree that `Error: Error:` is a bit ugly so I changed the splice thing with a lil' regex that'll work for all cases